doc/release/yarp_3_4/detect_clock_reset.md

```diff
@@ -0,0 +1,8 @@
+detect_clock_reset {#yarp_3_4}
+-----------------------
+
+### Libraries
+
+#### `os` 
+
+* If in a `yarp::os::NetworkClock` a clock reset is detected, unblock all the threads that are blocked waiting on a  `yarp::os::NetworkClock::delay` call on that network clock. A network clock reset is defined a jump in the past of the time published by the network clock port. This avoid that if `YARP_CLOCK` is set,  `yarp::os::PeriodicThread` silently stops to run if the network clock is reset. 
```

Comment:
The description here should also be updated...

Reply:
Thanks @drdanz :) 
Done in ab17f0f.